app/src/main/java/com/example/paetz/yacguide/MainActivity.java

```diff
@@ -29,4 +30,10 @@ public void enterTourbook(View v) {
         Intent intent = new Intent(this, TourbookActivity.class);
         startActivity(intent);
     }
+
+    public void enterTicklist(View v) {
+        Intent intent = new Intent(this, TourbookActivity.class);
+        intent.putExtra(IntentConstants.IS_TICKLIST_KEY, true);
```

Comment:
We should pass an enum or int type for tourbook-, säcke- and project-distinction. --> Done in follow-up ticket #52.